August 2015 bsides

GeordieMatt opened the pull request:
I had to go slightly off-piste for this one, because there were multiple mash steps in this batch. Let me know what you think! Also, Orange Blossom marked its last mash step as mashout, but I wasn't sure of the best way to mark that out (or if I needed it at all).

mattballaviva committed:
Add in the B-Sides brewed in August 2015.

mattballaviva committed:
Add the missing attenuation value to the Orange Blossom JSON.

samjbmason commented:
Thats an interesting one! I will have a think, I wonder if it is worth making all the mash steps an array of objects

samjbmason commented:
OK so I pushed an update that fixes this issue, all you need to do is rebase this branch with master and the tests should pass fine

sammdec commented on data/baby_saison.json:
```diff
@@ -0,0 +1,132 @@
+{
+  "name": "Baby Saison",
```
Could you add the B Sides prefix in at the end maybe e.g "Baby Saison - B-Sides"

sammdec commented on data/morags_mojito.json:
```diff
@@ -0,0 +1,116 @@
+{
+  "name": "Morag's Mojito",
```
Same thing here with the B-Sides thing

sammdec commented on data/orange_blossom.json:
```diff
@@ -0,0 +1,132 @@
+{
+  "name": "Orange Blossom",
```
Naming thing as before

samjbmason commented:
Apart from this couple of points and the rebase thing it all looks great :+1: thanks for contributing! :smile:

mattballaviva committed:
Add the B-Sides suffix to the new recipes.

GeordieMatt commented:
Just rebased and added those suffixes in. There might need to be a review of 'series' beers like Unleash the Yeast and IPA is Dead to make sure they're all marked the same. Happy to be on board!

samjbmason commented:
Yeah makes sense, I will make sure to check them, I dont think any have come through yet
